scipy/integrate/_tanhsinh.py

```diff
@@ -772,7 +772,7 @@ def _estimate_error(work, xp):
         d5 = log_e1 + xp.real(work.Sn)
         temp = xp.where(d1 > -xp.inf, d1 ** 2 / d2, -xp.inf)
         ds = xp.stack([temp, 2 * d1, d3, d4, d5])
```

Comment:
Oops, maybe I need to do:
```suggestion
        ds = xp.stack([temp, 2 * d1, d3, d4])
```
and below:
```python3
aerr = xp.maximum(xp.minimum(d1, xp.max(ds, axis=0)), d5)
```
?

I also need to update the description of the error estimate.

Reply:
Oh, yeah. I didn't think of that. That sounds right.

Reply:
I think I can use `xp.clip` to simplify, though.